### PR TITLE
Code Quality: Added hierarchical page navigation to UI Test app

### PR DESCRIPTION
**Pull request by 0x5bfa**
### Resolved / Related Issues

*No official tracker*

### Steps used to test these changes

1. Open Files UI Test app
2. Try the new look

image

### CODE REVIEW

**Comment by 0x5bfa** [comment]
I'll format C# later but looks fine.
@yaira2 this contains extended controls for MenuFlyoutItem controls.
Is it fine with you?

**Comment by mdtauk** [comment]
> I'll format C# later but looks fine. @yaira2 this contains extended controls for MenuFlyoutItem controls. Is it fine with you?

@yaira2 Those are just placeholder so I could have a second page to test if we can derive from the base controls ok.  They are an on-going project.

File: src/Files.App.Controls/MenuFlyoutItems/MenuFlyoutSubItemEx/MenuFlyoutSubItemEx.ThemeResources.xaml
Comment (yaira2) [inline src/Files.App.Controls/MenuFlyoutItems/MenuFlyoutSubItemEx/MenuFlyoutSubItemEx.ThemeResources.xaml:1]: Can you fix the formatting? It's possible the format command doesn't include the controls project.

Comment (0x5bfa) [reply src/Files.App.Controls/MenuFlyoutItems/MenuFlyoutSubItemEx/MenuFlyoutSubItemEx.ThemeResources.xaml:1]: Updated: https://github.com/files-community/Files/pull/15939

**Comment by yaira2** [comment]
@0x5bfa are you okay separating this PR from the menu item control?

**Comment by 0x5bfa** [comment]
Yeah

**Comment by 0x5bfa** [comment]
Ready for review.

**Review by yaira2 — APPROVED** [review]

**Pull request merged**